README.md

```diff
@@ -168,12 +168,7 @@ jobs:
         server: ftp.samkirkland.com
         username: myFtpUserName
         password: ${{ secrets.password }}
-        exclude: 
-          - .git*
-          - .git*/**
-          - '**/.git*/**'
-          - node_modules/**
-          - node_modules/**/*
+        exclude: '[".git*",".git*/**", "**/.git*/**", "node_modules/**", "node_modules/**/*"]'
```

Comment:
The double quotes aren't necessary in the list, those would be parsed as part of the glob pattern:
https://github.com/SamKirkland/FTP-Deploy-Action/blob/65c6a8f943bbf04faf9b59520a103318b418952e/src/parse.ts#L102

I assume that results in a failed match, though this should be tested.